src/ceph-crash.in

```diff
@@ -51,7 +51,7 @@ def post_crash(path):
         stderr = pr.communicate(input=f.read())
         rc = pr.wait()
         f.close()
-        if rc != 0:
+        if rc != 0 or stderr != "":
```

Comment:
just printing to stderr isn't an error, of course, so I don't know that calling this "failed" is really right, but it's a nit

Reply:
Agreed, but as explained in https://github.com/ceph/ceph/pull/42055/commits/8a89e0105d466d9a7659f6d5bd26f4e38d0d2758, I feel like it's better than nothing.